python-package/setup.py

```diff
@@ -46,5 +46,14 @@
       include_package_data=True,
       data_files=[('xgboost', LIB_PATH)],
       license='Apache-2.0',
-      classifiers=['License :: OSI Approved :: Apache Software License'],
+      classifiers=['License :: OSI Approved :: Apache Software License'
+                   "Development Status :: 5 - Production/Stable",
+                   "Operating System :: OS Independent",
+                   "Programming Language :: Python",
+                   "Programming Language :: Python :: 2",
+                   "Programming Language :: Python :: 2.7",
```

Comment:
We plan to drop support for Python 2.x by end of this year. Let's remove all references to Python 2.x.

Reply:
I have removed the reference to python2.